pkg/user/user.go

```
@@ -101,6 +101,32 @@ func (q *Client) Create(username, password string) (user *ForCreate, err error)
 	return
 }
 
+// CreateWithParams will create a user in Jenkins
+func (q *Client) CreateWithParams(data ForCreate) (user *ForCreate, err error) {
+	var (
+		payload io.Reader
+		code    int
+	)
+
+	userData, _ := json.Marshal(data)
```

Please don't ignore any errors.